model/metrics/system-metrics.yaml

```diff
@@ -0,0 +1,480 @@
+groups:
+  # General system attributes
+  - id: attributes.system
+    prefix: system
+    type: attribute_group
+    brief: "Describes System metric attributes"
+    attributes:
+      - id: device
+        type: string
+        brief: "The device identifier"
+        examples: ["(identifier)"]
+
+  # sytem.cpu.* metrics and attribute group
+  - id: attributes.system.cpu
+    prefix: system.cpu
+    type: attribute_group
+    brief: "Describes System CPU metric attributes"
+    attributes:
+      - id: state
+        type:
+          allow_custom_values: true
+          members:
+            - id: idle
+              value: 'idle'
+            - id: user
+              value: 'user'
+            - id: system
+              value: 'system'
+            - id: interrupt
+              value: 'interrupt'
+        brief: "The state of the CPU"
+        examples: ["idle", "interrupt"]
+      - id: cpu
+        type: int
+        brief: "The logical CPU number [0..n-1]"
+        examples: [1]
+
+  - id: metric.system.cpu.time
+    type: metric
+    metric_name: system.cpu.time
+    brief: "Seconds each logical CPU spent on each mode"
+    instrument: counter
+    unit: "s"
+    attributes:
+      - ref: system.cpu.state
+      - ref: system.cpu.cpu
```

Comment:
```suggestion
      - ref: system.cpu.logical.count
```

Reply:
Let's combine the discussion in a single thread https://github.com/open-telemetry/semantic-conventions/pull/89#discussion_r1282760528